browse/templates/stats/monthly_downloads.html

```diff
@@ -0,0 +1,179 @@
+{%- extends "stats/base.html" -%}
+
+{% block title %}Monthly Download Rates{% endblock %}
+{% block header_h1 %}<h1><a href="{{ url_for('.home') }}">arXiv.org</a> &gt; Monthly Downloads</h1>{% endblock %}
+
+{% block content %}
+<h2>arXiv Monthly Download Rates [<a title="download monthly download CSV data" href="{{ url_for('browse.stats', command='get_monthly_downloads') }}">CSV</a>]</h2>
+
+<style>
+  svg {
```

Comment:
I wonder if a jinja {% include monthly_downloads.css %} would work?
That would afford a nicer IDE experience for editing the CSS.